src/api/app/views/webui2/webui/groups/show.html.haml

```diff
@@ -0,0 +1,56 @@
+- @pagetitle = "Group #{@group.title}"
+
+.card
+  %div
+    %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible{ 'role': 'tablist' }
+      %li.nav-item
+        %a.active.nav-link.text-nowrap{ aria: { controls: 'reviews-in', selected: 'true' },
+        data: { toggle: 'tab' }, href: '#reviews-in', role: 'tab' }
+          Incoming Reviews
+      %li.nav-item
+        %a.nav-link.text-nowrap{ aria: { controls: 'requests-in' }, data: { toggle: 'tab' }, href: '#requests-in', role: 'tab' }
+          Incoming Requests
+      %li.nav-item
+        %a.nav-link.text-nowrap{ aria: { controls: 'all-requests' }, data: { toggle: 'tab' }, href: '#all-requests', role: 'tab' }
+          All Requests
+      %li.nav-item
+        %a.nav-link.text-nowrap{ aria: { controls: 'group-members' }, data: { toggle: 'tab' }, href: '#group-members', role: 'tab' }
+          Group Members
+
+  .card-body.tab-content
+    .tab-pane.show.active#reviews-in{ aria: { controls: 'reviews-in' }, role: 'tabpanel' }
+      %h3 Incoming Reviews
+      = render(partial: 'webui2/shared/requests_table', locals: { id: 'reviews_in_table', source_url: group_requests_path(@group) })
+
+    .tab-pane#requests-in{ aria: { controls: 'requests-in' }, role: 'tabpanel' }
+      %h3 Incoming Requests
+      = render(partial: 'webui2/shared/requests_table', locals: { id: 'requests_in_table', source_url: group_requests_path(@group) })
+
+    .tab-pane#all-requests{ aria: { controls: 'all-requests' }, role: 'tabpanel' }
+      %h3 All Requests
+      = render(partial: 'webui2/shared/requests_table', locals: { id: 'all_requests_table', source_url: group_requests_path(@group) })
+
+    .tab-pane#group-members{ aria: { controls: 'group-members' }, role: 'tabpanel' }
+      %h3 Group Members
+      %table.responsive.table.table-striped.table-bordered#group-members-table{ width: '100%' }
```

Comment:
`width: 100%` can be replaced by the Bootstrap class `w-100`

Reply:
> `width: 100%` can be replaced by the Bootstrap class `w-100`

Done.